src/OpenTelemetry/Metrics/Metric.cs

```diff
@@ -92,15 +93,10 @@ public sealed class Metric
             {
                 this.MetricType = MetricType.Histogram;
 
-                if (histogramBounds != null
-                    && histogramBounds.Length == 0)
-                {
-                    aggType = AggregationType.HistogramSumCount;
-                }
-                else
-                {
-                    aggType = AggregationType.Histogram;
-                }
+                // Use histogramBounds and histogramRecordMinMax to determine the aggregation type
+                aggType = histogramBounds != null && histogramBounds.Length == 0
+                    ? (histogramRecordMinMax ? AggregationType.HistogramSumCountMinMax : AggregationType.HistogramSumCount)
```

Comment:
the MetricType should be updated to reflect minmax, right?

Reply:
I'm confused as to what the purpose of `MetricType` is compared to using the `AggregationType`

Reply:
`MetricType` is public interface to allow Exporters to know what type of Metric they are receiving.

AggregationType is just internal to let aggregatos know "what to do with the incoming measurement".

Reply:
Okay, added in the latest commit. Let me know how that looks 👍🏻